### ISSUE TITLE
Another source for Windows precompiled binaries ?

### ISSUE THREAD

**Issue by Nandox7** [sensics/OSVR-RenderManager#384]
The page linked in the README  (http://osvr.github.io/using/) is still live but the links for Windows Runtime files or  SDK are dead.

For example: http://access.osvr.com/binary/osvr-sdk-installer
I have seen in the past the page dev put it back on, and I'll try to ping the owner in Reddit but it would be good if there is any other live source for this.

Trying to compile (and update as needed) all the OSVR/HDK components and currently stuck with OSVR-RenderManager.
Thanks

**Comment by rpavlik** [comment]
yeah, I gotta grab those and upload them. The backups of the binaries as well as the s3 credentials I used to grab them are on my personal laptop, which I don't have with me right now, so I can't help immediately, sorry. I'll try to find some time to stick at least one set of binaries up here.

**Comment by Nandox7** [comment]
No worries and thanks for helping on these and keeping this alive.
Today managed to have all the builds works in Linux, need to document it somehow.
Next will be windows, that should a completely different story.

**Comment by russell-taylor** [comment]
Hopefully this does the trick for the Windows RenderManager, including DirectMode.
[osvrRenderManager_0_6_43_unsigned.zip](https://github.com/sensics/OSVR-RenderManager/files/14297208/osvrRenderManager_0_6_43_unsigned.zip)

**Comment by rpavlik** [comment]
OK, so it looks like they did actually take down the S3, bummer.

I found a few things quick this morning, I put them in a release here: https://github.com/sensics/OSVR-RenderManager/releases/tag/v00_06_52

For some reason I seem to recall that the post-sensics build (labeled build 12) didn't work quite as well? but it is newer code. So not sure. But, that's why I uploaded so many versions.
